amazon/ionbenchmark/ion_benchmark_cli.py

```diff
@@ -94,6 +94,9 @@
 pypy = platform.python_implementation() == 'PyPy'
 if not pypy:
     import tracemalloc
+    import orjson
+else:
+    import json as orjson
```

Comment:
I think we should just throw if `orjson` is requested when using pypy, rather than silently replacing it.

Reply:
Yeah, I try to push other comments first to see if it pass the CI/CD, still working on it.